lungmask/mask.py

```diff
@@ -23,11 +23,15 @@
 def apply(image, model=None, force_cpu=False, batch_size=20, volume_postprocessing=True, noHU=False):
     if model is None:
         model = get_model('unet', 'R231')
-
-    inimg_raw = sitk.GetArrayFromImage(image)
-    directions = np.asarray(image.GetDirection())
-    if len(directions) == 9:
-        inimg_raw = np.flip(inimg_raw, np.where(directions[[0,4,8]][::-1]<0)[0])
+    
+    numpy_mode=isinstance(image, np.ndarray)
```

Comment:
can you please add spaces between the `=` for a consistent code style?

Reply:
Many thanks for the review! I will update with changes shortly.